pkg/infra/baseAgentMessages.go

```diff
@@ -46,15 +46,21 @@ func (a *Base) HandleStateHP(msg messages.StateHPMessage)
 func (a *Base) HandleStateIntendedFoodTaken(msg messages.StateIntendedFoodIntakeMessage) {}
 
 // Note: You can override this function depending on how you want to handle treaties.
-// This implementation automatically accepts treaties. You probably don't want to do this.
+// This implementation automatically rejects treaties. You probably don't want to do this.
 func (a *Base) HandleProposeTreaty(msg messages.ProposeTreatyMessage) {
```

Comment:
I might have missed something but why does the Handle have something inside by default? It can just be empty?

Also I just realized, this structure means that `a.activeTreaties[msg.TreatyID()] = treaty` has to be written by the agent teams (just a note)

Reply:
It could just be empty yeah, but it's more that this should be the 'default' behaviour - agents can still override this.

Yeah... I don't like that everyone would have to write `a.activeTreaties[msg.TreatyID()] = treaty`  either... not sure how to fix. Happy to take suggestions.

Reply:
Currently, it's the honest version of "reject every treaty that comes your way", so you let the sender know that you rejected. 
There's also the honest "accept every treaty that comes your way". 
These are the two extreme responses we probably want to test on honest runs. For more complicated functionality or for dishonest runs agent teams will override later I think.